enqueue/enqueue-bundle/0.3/etc/packages/test/enqueue.yaml

```diff
@@ -0,0 +1,6 @@
+enqueue:
+    transport:
+        default: 'null'
+        'null': ~
```

Comment:
Having a "null" object is very common and is, of course, a [well know design pattern](https://en.wikipedia.org/wiki/Null_Object_pattern); why are we trying to rename something that already describes exactly what the transport is?